libraries/ESP8266HTTPClient/examples/SSLCertificateVerification/SSLCertificateVerification.ino

```diff
@@ -0,0 +1,172 @@
+/**
+ * SSLCertificateVerification.ino
+ *
+ * Created on: 04.28.2017
+ *
+ * Copyright (c) 2017 Syed Salman Qadri. All rights reserved.
+ * This file is part of the ESP8266HTTPClient for Arduino.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ */
+
+#include <ESP8266WiFi.h>
+#include <ESP8266HTTPClient.h>
+#include <time.h>
+
+//WiFi
+const char wifissid[] = "WiFi";
+const char wifipsk[] = "Password";
+
+// This is a binary dump of the root CA.
+// You can also choose to load the cert via a file stream.
+// Here are the steps I did to dump the CA on my Mac:
```

Comment:
Also, what steps would users on other platforms take? Windows is probably the biggest question. Instructions for Linux will likely be the same, except for Mac-specific names ("Keychain Access", "Terminal").